tools/common.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+DOWNLOAD=download
+UNPACK=vendor
+INSTALL=install
+
+function update_dep_from_pypi {
+    local dep=$1
+    local version=$2
+
+    mkdir -p $DOWNLOAD
+
+    local expected=$DOWNLOAD/${dep}-${version}*
+
+    if [ ! -f $expected_file ]
+    then
+        # Remove any previous versions that might be hanging around
+        rm -f $DOWNLOAD/${dep}-[0-9]*
+
+        pip download \
+            -d $DOWNLOAD \
+            --no-deps \
+            --no-binary :all: \
+            ${dep}===${version}
```

Comment:
This creates a ${dep}-${version} directory. We should probably rename that to just ${dep} to make cleaning simpler and to make it easier to track changes in the code as dependencies are updated.